packages/in_app_purchase/in_app_purchase_storekit/pigeons/messages.dart

```diff
@@ -131,17 +131,17 @@ class SKProductMessage {
   const SKProductMessage(
       {required this.productIdentifier,
       required this.localizedTitle,
-      required this.localizedDescription,
       required this.priceLocale,
       required this.price,
+      this.localizedDescription,
       this.subscriptionGroupIdentifier,
       this.subscriptionPeriod,
       this.introductoryPrice,
       this.discounts});
 
   final String productIdentifier;
   final String localizedTitle;
-  final String localizedDescription;
+  final String? localizedDescription;
```

Comment:
Please put a comment on this referencing the issue, so someone doesn't change it back in the future thinking that this is just a mistake.